hack/crds-sync.sh

```diff
@@ -9,3 +9,4 @@ cp vendor/github.com/openshift/api/machineconfiguration/v1alpha1/zz_generated.cr
 #  These are MCO CRDs that live in other parts of the openshift/api, so the copies need to be more specific
 cp vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_80_machine-config_01_machineconfigurations*.crd.yaml install/.
 cp vendor/github.com/openshift/api/config/v1alpha1/zz_generated.crd-manifests/0000_10_config-operator_01_clusterimagepolicies*.crd.yaml install/.
+cp vendor/github.com/openshift/api/config/v1alpha1/zz_generated.crd-manifests/0000_10_config-operator_01_imagepolicies*.crd.yaml install/.
```

Comment:
No longer needed since #4597 [dropped this file][1] to move all the CRD management over to the API repo (so all you'll need is openshift/api#2036 once you have your controller here merged and ready to process instances of that CRD).

[1]: https://github.com/openshift/machine-config-operator/commit/5db1478b7d9aeecf343bfd91165686dec0c79c4f#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L59